src/fetchers/stats-fetcher.js

```diff
@@ -226,6 +231,7 @@ const fetchStats = async (
   include_merged_pull_requests = false,
   include_discussions = false,
   include_discussions_answers = false,
+  year = undefined,
```

Comment:
Do you really need to explicitly spicify `= undefined` there? I think that behavior of the code will be the same without it,

```suggestion
  year,
```